polling_stations/templates/fragments/cancelled_election.html

```diff
@@ -14,12 +14,11 @@ <h2>{% trans "Cancelled Election" %}</h2>
     {% else %}
         <p>{% blocktrans with cancelled_election_name=cancelled_election.name %}The poll for <strong>{{ cancelled_election_name }}</strong> has been cancelled.{% endblocktrans %}</p>
     {% endif %}
+    <p>To learn more, please visit <a href="https://whocanivotefor.co.uk/elections/{{ cancelled_election.slug }}/">Who Can I Vote For?</a></p>
 {% endif %}
 
 {% if cancelled_election.metadata.cancelled_election.detail %}
-    {% filter markdown %}
-        {{ cancelled_election.metadata.cancelled_election.detail }}
-    {% endfilter %}
+    {% filter markdown %}{{ cancelled_election.metadata.cancelled_election.detail }}{% endfilter %}
```

Comment:
If your pipe solution works maybe it's better as it can't be regressed by someone deciding to split this line into three again?